docs/Reference/API-Methods.md

```diff
@@ -1042,17 +1042,26 @@ None
 
 ### `eth_gasPrice`
 
-Returns the current gas unit price, in wei. It's the hexadecimal equivalent of the price specified
-for the [`--min-gas-price`](CLI/CLI-Syntax.md#min-gas-price) command line option when the node
-started, or the default minimum gas price.
+Returns a percentile gas unit price for the most recent blocks, in Wei. By default, 
+the last 100 blocks are examined and the 50th percentile gas unit price (that is, the median value)
+is returned.
+
+If there are no blocks, the value for [`--min-gas-price`](CLI/CLI-Syntax.md#min-gas-price) is returned.
+The value returned is restricted to values between [`--min-gas-price`](CLI/CLI-Syntax.md#min-gas-price)
+and [`--api-gas-price-max`](CLI/CLI-Syntax.md#api-gas-price-max). By default, 1000 Wei and
```

Comment:
@shemnon - I've included the default min-gas-price here (1000) not the 100 from your [original PR](https://github.com/hyperledger/besu/pull/1563). Flagging in case I've got something wrong.

Reply:
I confirm, this may have been a typo. Min gas price default is `Wei.of(1000)`

Reply:
Yea, typo.  1000 is a fine minimum.  Mostly a flag that there have been no TXes.